### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by IamNorma**
Refactor classes to include mathable and collectable modules.

### CODE REVIEW

File: lib/team.rb [hunk 1]
```diff
@@ -1,15 +1,13 @@
...
+
   @@all = []
   attr_reader :team_id, :franchise_id, :team_name,
               :abbreviation, :stadium, :link
 
-  def self.from_csv(file_path)
-    csv = CSV.read("#{file_path}", headers: true, header_converters: :symbol )
-    @@all = csv.map{|row| Team.new(row)}
-  end
-
```
Comment (whitneykidd) [inline lib/team.rb:12]: So glad we could move this method to a module instead 💯

File: lib/team_stats.rb [hunk 2]
```diff
@@ -39,9 +39,6 @@ def average_win_percentage(team_id)
     games_played = games_played_in(team_id)
     games_won = games_played.select{|game| game.result == "WIN"}.length
     percent(games_won, games_played)
-    # win_ratio = games_won.length.to_f / games_played.length.to_f
-    # win_percentage = win_ratio.round(2)
-    # win_percentage
```
Comment (kiverso) [inline lib/team_stats.rb]: I like how the module let us shorten this method so much.

File: lib/collectable.rb [hunk 3]
```diff
@@ -0,0 +1,8 @@
+require 'CSV'
+
+module Collectable
```
Comment (dborski) [inline lib/collectable.rb:3]: This module was definitely a great opportunity to reduce duplicating code.

File: lib/league_stats.rb [hunk 4]
```diff
@@ -46,26 +45,23 @@ def worst_offense
...
-     total_games = games.length.to_f
+     total_games = games.length
      goals = games.sum {|games| games.goals}
-     (goals / total_games)
-    end
-    highest_scoring = away_team_records.max_by do |team, average_goals|
-     average_goals
+     average(goals, total_games)
     end
+    highest_scoring = away_team_records.max_by { |team, avg_goals| avg_goals }
```
Comment (kiverso) [inline lib/league_stats.rb:52]: another method that is much shorter and easier to read thanks to mathable module. nice work.

**Pull request merged**